src/itemTracker/dungeonTracker.js

```diff
@@ -311,6 +318,45 @@ export default class DungeonTracker extends React.Component{
                     }
                     
                 </Row>
+                {this.props.entranceRando !== "None" &&
+                    <Row>
```

Comment:
Make this row a noGutter (see other rows in the dungeon tracker component), and consider messing with some styling (I can give you tips and ideas, but use dev tools to identify what changes may be needed)

Reply:
The biggest (and most obvious) thing there is too much spacing between the name and the entered marker